### DISCUSSION TITLE
What's the deal w/ Octane?

### DISCUSSION THREAD

**Discussion by longlho** [ember-intl/ember-intl#1529] [Q&A]
Seems like we're on an older way of doing things (non-native class, manual `computed`, event emitting seems different). @jasonmit can u provide some context since idk much about Ember?

**Comment by jasonmit** [comment]
I've been out of the Ember community myself for two years, beyond the maintenance aspects of this lib, so I don't know the current status of octane when it comes to addon support.  But some context, this ties back to the conversation around dropping legacy versions.  Since we're targeting older versions, we had to stay on the traditional Ember path/non-octane which works with legacy and current.

Last I knew, which is dated info, native classes were hung up on decorator support and that was the last I knew of it.  @buschtoens is way more involved on that front than me and can give a proper update :)

**Reply by longlho** [reply]
Seems like some stuff are getting deprecated and docs seem to point to octane that's why I'm confused. More reason to drop older version support...
